### PR TITLE
combine and simplify sq and exports

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr consolidates the saved queries and exports docs so its clearer how both work together. it also removes the comparison table and adds a 'configure exports' section in teh saved queries page, making sure they flow correctly. 

[notion project](https://www.notion.so/dbtlabs/Consolidate-Save-queries-and-Exports-document-e5cc1a7ccd9541089a57c9d9ea5008f4?pvs=4)

### CODE REVIEW

File: website/docs/docs/use-dbt-semantic-layer/exports.md [hunk 1]
```diff
@@ -4,88 +4,41 @@ description: "Use exports to write tables to the data platform on a schedule."
...
+## Exports in development
 
 You can run an export in your development environment using your development credentials if you want to test the output of the export before production. You can use the following command to run exports in the dbt Cloud CLI:
 
 ```bash
 dbt sl export
 ```
 
-The following table lists the options for `dbt sl export` command:  
+The following table lists the options for `dbt sl export` command, using the `--` flag to specify the parameters:
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/exports.md]:
```suggestion
The following table lists the options for `dbt sl export` command, using the `--` flag prefix to specify the parameters:  
```

File: website/docs/docs/build/saved-queries.md [hunk 2]
```diff
@@ -65,73 +113,65 @@ saved_queries:
...
-| `exports::config::schema`   | String   | Optional    | The schema for creating the table or view. This option cannot be used for caching.   |
-| `exports::config::alias`  | String     | Optional    | The table alias used to write to the table or view.  This option cannot be used for caching.  |
+## Run exports
 
-</VersionBlock> 
+Once you've configured exports, you can now take things a step further by running exports to automatically write saved queries within your data platform using [dbt Cloud's job scheduler](/docs/deploy/job-scheduler). This feature is only available with the [dbt Cloud's Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl).
 
-All metrics in a saved query need to use the same dimensions in the `group_by` or `where` clauses.
-When using the `Dimension` object, prepend the semantic model name, for example `Dimension('user__ds')`
+For more information on how to run exports, refer to the [Exports](/docs/use-dbt-semantic-layer/exports)documentation.
```
Comment (mirnawong1) [inline website/docs/docs/build/saved-queries.md]:
```suggestion
For more information on how to run exports, refer to the [Exports](/docs/use-dbt-semantic-layer/exports) documentation.
```

File: website/docs/docs/build/saved-queries.md [hunk 3]
```diff
@@ -36,19 +84,19 @@ saved_queries:
       cache:
```
Comment (mirnawong1) [inline website/docs/docs/build/saved-queries.md]: Add file name semantic model

File: website/docs/docs/build/saved-queries.md [hunk 4]
```diff
@@ -6,19 +6,67 @@ sidebar_label: "Saved queries"
...
-| **Integration**     | Must have the dbt Semantic Layer configured in your dbt project.<br /><br />Tightly integrated with the [MetricFlow Server](/docs/use-dbt-semantic-layer/sl-architecture#components) and dbt Cloud's job scheduler. | Integrated into the dbt <Term id="dag" /> and managed alongside other dbt nodes. |
-| **Configuration**   | Defined within the `saved_queries` configuration. Set up within the dbt Cloud environment and job scheduler settings. | Defined in YAML format within dbt project files.     |
+</VersionBlock>
+
+All metrics in a saved query need to use the same dimensions in the `group_by` or `where` clauses.
+Use the semantic model name prefix with the Dimension object, like `Dimension('user__ds')`.
+
+## Configure saved query
+
+Use saved queries to define and manage common Semantic Layer queries in YAML, including metrics and dimensions. Saved queries enable you to organize and reuse common MetricFlow queries within dbt projects. For example, you can group related metrics together for better organization, and include commonly used dimensions and filters.
```
Comment (mirnawong1) [inline website/docs/docs/build/saved-queries.md:68]: Include dbt_project.yml config (saved-queries) blurb

File: website/docs/docs/build/saved-queries.md [hunk 5]
```diff
@@ -6,19 +6,67 @@ sidebar_label: "Saved queries"
...
-| **Integration**     | Must have the dbt Semantic Layer configured in your dbt project.<br /><br />Tightly integrated with the [MetricFlow Server](/docs/use-dbt-semantic-layer/sl-architecture#components) and dbt Cloud's job scheduler. | Integrated into the dbt <Term id="dag" /> and managed alongside other dbt nodes. |
-| **Configuration**   | Defined within the `saved_queries` configuration. Set up within the dbt Cloud environment and job scheduler settings. | Defined in YAML format within dbt project files.     |
+</VersionBlock>
+
+All metrics in a saved query need to use the same dimensions in the `group_by` or `where` clauses.
+Use the semantic model name prefix with the Dimension object, like `Dimension('user__ds')`.
+
```
Comment (mirnawong1) [inline website/docs/docs/build/saved-queries.md:65]: @Jstein77 do we need add 'saved-queries' to the [config property doc](https://docs.getdbt.com/reference/resource-properties/config) or [enabled](https://docs.getdbt.com/reference/resource-configs/enabled)?

File: website/docs/docs/build/saved-queries.md [hunk 6]
```diff
@@ -6,19 +6,67 @@ sidebar_label: "Saved queries"
...
-| **Integration**     | Must have the dbt Semantic Layer configured in your dbt project.<br /><br />Tightly integrated with the [MetricFlow Server](/docs/use-dbt-semantic-layer/sl-architecture#components) and dbt Cloud's job scheduler. | Integrated into the dbt <Term id="dag" /> and managed alongside other dbt nodes. |
-| **Configuration**   | Defined within the `saved_queries` configuration. Set up within the dbt Cloud environment and job scheduler settings. | Defined in YAML format within dbt project files.     |
+</VersionBlock>
+
+All metrics in a saved query need to use the same dimensions in the `group_by` or `where` clauses.
+Use the semantic model name prefix with the Dimension object, like `Dimension('user__ds')`.
```
Comment (Jstein77) [inline website/docs/docs/build/saved-queries.md]: you actually need to use the entity name prefix not the semantic model name prefix

File: website/docs/docs/build/saved-queries.md [hunk 7]
```diff
@@ -6,19 +6,67 @@ sidebar_label: "Saved queries"
...
-| **Integration**     | Must have the dbt Semantic Layer configured in your dbt project.<br /><br />Tightly integrated with the [MetricFlow Server](/docs/use-dbt-semantic-layer/sl-architecture#components) and dbt Cloud's job scheduler. | Integrated into the dbt <Term id="dag" /> and managed alongside other dbt nodes. |
-| **Configuration**   | Defined within the `saved_queries` configuration. Set up within the dbt Cloud environment and job scheduler settings. | Defined in YAML format within dbt project files.     |
+</VersionBlock>
+
+All metrics in a saved query need to use the same dimensions in the `group_by` or `where` clauses.
```
Comment (Jstein77) [inline website/docs/docs/build/saved-queries.md]: I would update this slightly for clarity. "If you use multiple metrics in a saved query, then you will only be able to refrence the common dimensions these metrics share in the `group_by` or `where`

File: website/docs/docs/build/saved-queries.md (hunk 5 above)
Comment (Jstein77) [reply website/docs/docs/build/saved-queries.md:65]: Yes!

**Review by Jstein77 — APPROVED** [review]
LGTM! Left a few comments.

File: website/docs/docs/build/saved-queries.md (hunk 7 above)
Comment (mirnawong1) [reply website/docs/docs/build/saved-queries.md]: "If you use multiple metrics in a saved query, then you will only be able to reference the common dimensions these metrics share in the `group_by` or `where` clauses. Use the entity name prefix with the Dimension object, like `Dimension('user__ds')`."

File: website/docs/docs/build/saved-queries.md [hunk 8]
```diff
@@ -6,19 +6,66 @@ sidebar_label: "Saved queries"
...
-| **Configuration**   | Defined within the `saved_queries` configuration. Set up within the dbt Cloud environment and job scheduler settings. | Defined in YAML format within dbt project files.     |
+| Parameter | Type    | Required | Description    |
+|-------|---------|----------|----------------|
+| `name`       | String    | Required     | Name of the saved query object.          |
+| `description`     | String      | Required     | A description of the saved query.     |
+| `label`     | String      | Required     | The display name for your saved query. This value will be shown in downstream tools.    |
+| `config`     | String      | Required     | A config section for any parameters specifying the saved query.   |
+| `config::cache`     | String      | Optional     |  A boolean to specify if a saved query should be used to populate the cache. Accepts `true` or `false`. Defaults to `false` |
```
Comment (mirnawong1) [inline website/docs/docs/build/saved-queries.md]: is this really required or optional? m

File: website/docs/docs/build/saved-queries.md [hunk 9]
```diff
@@ -36,27 +84,42 @@ saved_queries:
...
+      - name: my_export
+        config:
+          alias: my_export_alias
+          export_as: table
+          schema: my_export_schema_name
 ```
 </File>
 
-</VersionBlock> 
+To enable saved queries at the project level, you can set the `saved-queries` configuration in the [`dbt_project.yml` file](/reference/dbt_project.yml). This saved you time in configuring saved queries in each file:
```
Comment (mirnawong1) [inline website/docs/docs/build/saved-queries.md]:
```suggestion
To enable saved queries at the project level, you can set the `saved-queries` configuration in the [`dbt_project.yml` file](/reference/dbt_project.yml). This saves you time in configuring saved queries in each file:
```

File: website/docs/docs/build/saved-queries.md [hunk 10]
```diff
@@ -6,25 +6,73 @@ sidebar_label: "Saved queries"
...
+<!-- For versions 1.8 and higher -->
+<VersionBlock firstVersion="1.8">
 
-All metrics in a saved query need to use the same dimensions in the `group_by` or `where` clauses. The following is an example of a saved query:
+| Parameter | Type    | Required | Description    |
+|-------|---------|----------|----------------|
+| `name`       | String    | Required     | Name of the saved query object.          |
+| `description`     | String      | Required     | A description of the saved query.     |
+| `label`     | String      | Required     | The display name for your saved query. This value will be shown in downstream tools.    |
+| `config`     | String      | Required     |  Use the [config](/reference/resource-properties/config) property to specify configurations for your saved query. Supports [meta](/reference/resource-configs/meta), [group](/reference/resource-configs/group), and [enabled](/reference/resource-configs/enabled) configurations.   |
```
Comment (mirnawong1) [inline website/docs/docs/build/saved-queries.md]:
```suggestion
| `config`     | String      |  Optional     |  Use the [config](/reference/resource-properties/config) property to specify configurations for your saved query. Supports [meta](/reference/resource-configs/meta), [group](/reference/resource-configs/group), and [enabled](/reference/resource-configs/enabled) configurations.   |
```

File: website/docs/docs/build/saved-queries.md (hunk 8 above)
Comment (mirnawong1) [reply website/docs/docs/build/saved-queries.md]: Jordan confirmed it's optional

File: website/docs/docs/build/saved-queries.md [hunk 11]
```diff
@@ -79,62 +142,53 @@ saved_queries:
...
-</VersionBlock> 
+Once you've configured exports, you can now take things a step further by running exports to automatically write saved queries within your data platform using [dbt Cloud's job scheduler](/docs/deploy/job-scheduler). This feature is only available with the [dbt Cloud's Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl).
 
-All metrics in a saved query need to use the same dimensions in the `group_by` or `where` clauses.
-When using the `Dimension` object, prepend the semantic model name, for example `Dimension('user__ds')`
+For more information on how to run exports, refer to the [Exports](/docs/use-dbt-semantic-layer/exports) documentation.
 
 ## Related docs
 
 - [Exports](/docs/use-dbt-semantic-layer/exports)
```
Comment (mirnawong1) [inline website/docs/docs/build/saved-queries.md]:
```suggestion
- Run [exports](/docs/use-dbt-semantic-layer/exports)
```

File: website/docs/docs/build/saved-queries.md [hunk 12]
```diff
@@ -79,62 +142,53 @@ saved_queries:
...
 
-All metrics in a saved query need to use the same dimensions in the `group_by` or `where` clauses.
-When using the `Dimension` object, prepend the semantic model name, for example `Dimension('user__ds')`
+For more information on how to run exports, refer to the [Exports](/docs/use-dbt-semantic-layer/exports) documentation.
 
 ## Related docs
 
-- [Exports](/docs/use-dbt-semantic-layer/exports)
-- [Set up the dbt Semantic Layer](/docs/use-dbt-semantic-layer/setup-sl)
+- Run [exports](/docs/use-dbt-semantic-layer/exports)
```
Comment (mirnawong1) [inline website/docs/docs/build/saved-queries.md]:
```suggestion
```

File: website/docs/docs/use-dbt-semantic-layer/exports.md [hunk 13]
```diff
@@ -4,88 +4,41 @@ description: "Use exports to write tables to the data platform on a schedule."
 sidebar_label: "Write queries with exports"
 ---
 
-The exports feature in the dbt Semantic Layer enhances the [saved queries](/docs/build/saved-queries) by allowing you to write commonly used queries directly within your data platform.
+Exports enhances the [saved queries](/docs/build/saved-queries) by automatically run saved queries and write them within your data platform. Saved queries are a way to save and reuse commonly used queries in MetricFlow, exports takes this functionality a step further by:
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/exports.md]:
```suggestion
Exports enhances [saved queries](/docs/build/saved-queries) by running your saved queries and writing the output to a table or view within your data platform. Saved queries are a way to save and reuse commonly used queries in MetricFlow, exports take this functionality a step further by:
```

**Pull request merged**